info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-23T02:30:15.854Z"}
info: Connected to the database. {"timestamp":"2022-02-23T02:30:15.884Z"}
error: 
    status - 500 
    message - Error: password is invalid
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:122:19)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T02:30:25.162Z"}
error: 
    status - 500 
    message - Error: password is invalid
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:122:19)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T04:06:39.376Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected token } in JSON at position 75
    at JSON.parse (<anonymous>)
    at parse (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\body-parser\lib\types\json.js:89:19)
    at C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\body-parser\lib\read.js:121:18
    at invokeCallback (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\raw-body\index.js:224:16)
    at done (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (node:events:402:35)
    at endReadableNT (node:internal/streams/readable:1343:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T04:07:12.726Z"}
error: 
    status - 500 
    message - Error: data must be a string or Buffer and hash must be a string
    at Object.compareSync (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\bcrypt\bcrypt.js:170:15)
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:107:48)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T04:07:23.563Z"}
error: 
    status - 500 
    message - Error: password is invalid
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:122:19)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T04:07:36.159Z"}
error: 
    status - 500 
    message - Error: password is invalid
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:122:19)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T04:07:46.642Z"}
error: 
    status - 500 
    message - Error: password is invalid
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:122:19)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T04:07:58.793Z"}
error: 
    status - 500 
    message - Error: password is invalid
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:122:19)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T04:11:24.676Z"}
error: 
    status - 500 
    message - Error: password is invalid
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:122:19)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T04:12:11.337Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-23T06:04:41.260Z"}
info: Connected to the database. {"timestamp":"2022-02-23T06:04:41.284Z"}
error: 
    status - 500 
    message - Error: password is invalid
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:122:19)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T06:06:59.669Z"}
error: 
    status - 500 
    message - Error: password is invalid
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:122:19)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T06:07:02.085Z"}
error: 
    status - 500 
    message - Error: password is invalid
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:122:19)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T06:07:20.892Z"}
error: 
    status - 500 
    message - ValidationError: "firstname" is required 
    url - /api/v1/users/userregister 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-23T06:07:45.633Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-23T06:09:40.206Z"}
info: Connected to the database. {"timestamp":"2022-02-23T06:09:40.248Z"}
error: 
    status - 500 
    message - Error: password is invalid
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:122:19)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T06:12:53.087Z"}
error: 
    status - 500 
    message - Error: password is invalid
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:122:19)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T06:13:05.663Z"}
error: 
    status - 500 
    message - Error: password is invalid
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:122:19)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T06:13:11.442Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-23T07:31:12.773Z"}
info: Connected to the database. {"timestamp":"2022-02-23T07:31:12.808Z"}
error: 
    status - 500 
    message - Error: password is invalid
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:122:19)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T08:20:17.671Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-23T08:21:36.795Z"}
info: Connected to the database. {"timestamp":"2022-02-23T08:21:36.830Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-23T08:32:41.112Z"}
info: Connected to the database. {"timestamp":"2022-02-23T08:32:41.148Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-23T08:33:54.898Z"}
info: Connected to the database. {"timestamp":"2022-02-23T08:33:54.931Z"}
error: 
    status - 500 
    message - Error: password is invalid
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:122:19)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T08:34:17.139Z"}
error: 
    status - 500 
    message - Error: password is invalid
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:122:19)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T08:34:29.445Z"}
error: 
    status - 500 
    message - Error: password is invalid
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:122:19)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T08:34:39.021Z"}
error: 
    status - 500 
    message - Error: password is invalid
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:122:19)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T08:37:35.010Z"}
error: 
    status - 500 
    message - Error: password is invalid
    at _callee3$ (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\build\services\user.service.js:122:19)
    at tryCatch (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\Vaibhav Ramu Dharmik\project\fundoo-Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-23T08:39:35.451Z"}
